hindi/art-of-living/art-of-living.md

```diff
@@ -3,4 +3,11 @@ and humanitarian movement engaged in stress-management and service initiatives.
 The organization operates globally in 154 countries and has touched the lives
 of over 370 million people.
 
-To know more, kindly visit this [link](http://www.artofliving.org/in-en)
\ No newline at end of file
+To know more, kindly visit this [link](http://www.artofliving.org/in-en)
+
+It is also considered to be the world's largest volunteer-based NGO by India today newspaper. [link](http://indiatoday.intoday.in/education/story/sri-sri-ravi-shankars-59th-birthday-10-interesting-facts/1/437746.html)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.